client-test/tests/client.rs

```diff
@@ -49,8 +48,16 @@ use {
         time::{Duration, Instant},
     },
     systemstat::Ipv4Addr,
+    tungstenite::connect,
 };
 
+const TEST_ACCOUNT_SUBSCRIPTION_PORT: u16 = rpc_port::DEFAULT_RPC_PUBSUB_PORT + 1;
```

Comment:
I fear this block of preallocated ports won't be updated when the next test is added to this file, as it's so easy to copy-paste an existing one.

Instead how about making the ports dynamic, like this:
```
use std::sync::atomic::{AtomicU16, Ordering};
    
static NEXT_RPC_PUBSUB_PORT: AtomicU16 = std::sync::atomic::AtomicU16::new(rpc_port::DEFAULT_RPC_PUBSUB_PORT);
    
fn pubsub_addr() -> SocketAddr {
    SocketAddr::new(
        IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)),
        NEXT_RPC_PUBSUB_PORT.fetch_add(1, Ordering::Relaxed)
    )  
}
```

Reply:
oh yeah. I used a similar way at the beginning but it failed when I use nextest to test it. 😢  Maybe they do something wrong. just tested it with normal cargo test and worked fine

Reply:
I opened an issue to nextest project. I think the way that you suggested is the best. Before nextest happen, I would like to use this solution. modified at [2d53131](https://github.com/solana-labs/solana/pull/29255/commits/2d531313f1dca4e21dac690f6c12adf6b2bfbb24) and [a841261](https://github.com/solana-labs/solana/pull/29255/commits/a841261e89acbd214cf0e17aceb1308ef48fa4bb)